app/models/post.rb

```diff
@@ -7,6 +7,12 @@ class Post < ApplicationRecord
 
   scope :get_5_comments, ->(post) { post.comments.order('created_at desc').limit(5) }
 
+  validates :title, presence: true
+  validates :author, presence: true
+  validates :title, length: { in: 1..250 }
+  validates :likes_counter, numericality: { only_integer: true }, comparison: { greater_than_or_equal_to: 0 }
```

Comment:
![image](https://github.com/emhamza/my-blog/assets/93039359/0ce8653b-8d2e-4963-8d4c-1fc9845c9b84)

- [ ] It would be nice if you follow the project requirements to make tests for all the validations because you have missed one validation test which is that the title length should not be more than 250 character 😨

Reply:
@HAMMAS-SALEEM Noted!
😄